### PR TITLE
Dev cow minigame

### PR DESCRIPTION
**Pull request by rujoota**
@jensvdh @worldofbalance/cards-of-wild Merging working cow with main server

### CODE REVIEW

**Comment by jensvdh** [comment]
There are still a few merge conflicts here.

**Comment by jensvdh** [comment]
Also I'm seeing lot of mixed tabs and spaces. Please indent using spaces only. Every tab should convert into 4 space characters.

File: src/lby/MiniGameServers.java [hunk 1]
```diff
@@ -57,8 +57,8 @@ private void initMiniGames() {
         game.setAsMultiPlayerGame("mini_game_server_jar/Speed_Server.jar", 20039);
         miniGames.put(game.getName(), game);
         
-        game = new MiniGame("Cards of the Wild");
+        /*game = new MiniGame("Cards of the Wild");
         game.setAsMultiPlayerGame("mini_game_server_jar/Cards_Server.jar", 20038);
```
Comment (niculistana) [inline src/lby/MiniGameServers.java:64]: @jensvdh this commit makes it so that the .jar file is not going to be built.

**Comment by niculistana** [comment]
@jensvdh It looks like @rujoota is combining the GameServer from the previous COW repository to the GameServer from the shared repository.

**Comment by niculistana** [comment]
Hi @rujoota, it seems that https://github.com/worldofbalance/wob-server/tree/dev-cow-minigame is 
31 commits behind development, and 11 commits ahead @jensvdh suggests that you guys pull code from development and handle the pull request before it gets really big. 

I also suggest that you pull changes from dev-rr-before-merge and dev-sdv-minigame as wel. Let me know if you need help!

**Comment by rujoota** [comment]
Yes, its behind...I am working on creating independent server and making it stable. That I guess this pull request won't be necessary.

**Comment by niculistana** [comment]
@rujoota I see, no worries! Will take a look at the other pull request. I will close this one now.

**Pull request closed without merging**